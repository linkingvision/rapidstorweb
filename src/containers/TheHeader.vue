<template>
	<CHeader with-subheader>
		<!-- 导航栏 -->
		<CSubheader class="px-3">
			<div class="dao_dian">.</div>
			<CBreadcrumbRouter class="border-0 mb-0">
				
			</CBreadcrumbRouter>
		</CSubheader>
	</CHeader>
</template>

<script>

export default {
  name: 'TheHeader'
}
</script>
<style scoped>
.c-header{
	min-height: 46px;
}
.c-dark-theme .c-subheader {
    background: #475058;
    border-bottom: 1px solid #5B5F63;
}
.c-dark-theme .c-header {
    background: #2c2c34;
    border-bottom: 0;
}
.breadcrumb{
	margin: 0;
	border: 0;
}
.dao_dian{
	text-align: center;
	line-height: 25px;
	font-size: 30px;
	color: #55C3EB;
}
</style>
