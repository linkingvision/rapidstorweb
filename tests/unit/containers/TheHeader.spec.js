import CoreuiVue from '@coreui/vue'
import TheHeader from '@/containers/TheHeader'
import Vuex from 'vuex'
import { shallowMount, createLocalVue } from '@vue/test-utils';

const localVue = createLocalVue()
localVue.use(CoreuiVue)

localVue.use(Vuex)

const store = new Vuex.Store({
  state: {
    sidebarShow: 'responsive',
    sidebarMinimize: false,
    asideShow: false,
    darkMode: false
  }
})

describe('TheHeader.vue', () => {
  test('renders correctly', () => {
    const wrapper = shallowMount(TheHeader, { localVue, store })
    expect(wrapper.element).toMatchSnapshot()
  })
})