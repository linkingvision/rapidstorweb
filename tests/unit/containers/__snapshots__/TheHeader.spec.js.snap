// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheHeader.vue renders correctly 1`] = `
<cheader-stub
  colorscheme="light"
  fixed="true"
  tag="header"
  withsubheader="true"
>
  <ctoggler-stub
    class="ml-3 d-lg-none"
    inheader="true"
    tag="button"
  />
   
  <ctoggler-stub
    class="ml-3 d-md-down-none"
    inheader="true"
    tag="button"
  />
   
  <cheaderbrand-stub
    activeclass="router-link-active"
    alt="CoreUI Logo"
    class="mx-auto d-lg-none"
    event="click"
    exactactiveclass="router-link-exact-active"
    height="46"
    routertag="a"
    src="img/brand/coreui-vue-pro-logo.svg"
    tag="div"
    target="_self"
    width="190"
  />
   
  <cheadernav-stub
    class="d-md-down-none mr-auto"
  >
    <cheadernavitem-stub
      class="px-3"
    >
      <cheadernavlink-stub
        activeclass="router-link-active"
        event="click"
        exactactiveclass="router-link-exact-active"
        routertag="a"
        target="_self"
        to="/dashboard"
      >
        
        Dashboard
      
      </cheadernavlink-stub>
    </cheadernavitem-stub>
  </cheadernav-stub>
   
  <cheadernav-stub>
    <cheadernavitem-stub
      class="px-3"
    >
      <button
        class="c-header-nav-btn"
      >
        <cicon-stub
          name="cil-moon"
        />
      </button>
    </cheadernavitem-stub>
  </cheadernav-stub>
   
  <csubheader-stub
    class="px-3"
    tag="div"
  >
    <cbreadcrumbrouter-stub
      class="border-0 mb-0"
    />
  </csubheader-stub>
</cheader-stub>
`;
